vars.yml

```diff
@@ -0,0 +1,12 @@
+ospSubnet: <Subnet for machines>
+externalNetwork: <Network cluster is on>
+apiFIP: <FIP for controlplane>
+ingressFIP: <FIP for dataplane>
+bootstrapFIP: <FIP for bootstrap access>
+custID: <Estate API ID for cluster>
+baseDomain: <Base domain for OSP region>
+rhcosImage: <rhcos image present in OSP project>
+externalDNS: <List of 2 DNS servers>
+apiAllowedSources: "0.0.0.0/0"
+ingressAllowedSources: "0.0.0.0/0"
+sshAllowedSources: "0.0.0.0/0"
```

Comment:
Do we have any safer defaults we could use here?

Reply:
The control plane and dataplane should be open to all sources by default but I've restricted the ssh sources to office and VPN networks only